parastell/magnet_coils.py

```diff
@@ -14,7 +14,91 @@
 
 
 class MagnetSet(object):
-    """An object representing a set of modular stellarator magnet coils.
+    """A minimum viable class which can be used to build a DAGMC model with
+    parastell utilizing pre-defined geometry
+
+    Arguments:
+        geom_filename (path): Path to the predefined magnet geometry
+        logger (object): logger object (optional, defaults to None). If no
+            logger is supplied, a default logger will be instantiated.
+
+    Optional Attributes
+        mat_tag (str): DAGMC material tag to use for magnets in DAGMC
+            neutronics model (defaults to 'magnets').
+    """
+
+    def __init__(self, geom_filename, logger=None, **kwargs):
+        self.logger = logger
+        geom_path = Path(geom_filename).resolve()
+        self.geom_filename = geom_path.name
+        self.export_dir = geom_path.parent
+        self.mat_tag = "magnets"
+        for name in kwargs.keys() & ("mat_tag"):
+            self.__setattr__(name, kwargs[name])
+
+    @property
+    def logger(self):
+        return self._logger
+
+    @logger.setter
+    def logger(self, logger_object):
+        self._logger = log.check_init(logger_object)
+
+    def import_geom_cubit(self):
+        """Import geom file for magnet set into Coreform Cubit."""
+        first_vol_id = 1
+        if cubit_io.initialized:
+            first_vol_id += cubit.get_last_id("volume")
+
+        last_vol_id = cubit_io.import_geom_to_cubit(
+            self.geom_filename, self.export_dir
+        )
+
+        self.volume_ids = list(range(first_vol_id, last_vol_id + 1))
```

Comment:
Are we sure that the volume ids are contiguous? I think there is a cubit command to ensure this.